tests/factories/models.py

```diff
@@ -35,6 +39,20 @@ class Meta:
     email = factory.SubFactory(factory="tests.factories.models.EmailFactory")
 
 
+class FirefoxAccountFactory(BaseSQLAlchemyModelFactory):
+    class Meta:
+        model = models.FirefoxAccount
+
+    fxa_id = factory.LazyFunction(lambda: uuid4().hex)
+    primary_email = factory.SelfAttribute("email.primary_email")
```

Comment:
👌 
Shall we add a comment that this simulates the DB relationship?

Reply:
I'm confused by my own comment...

I didn't know about `SelfAttribute` and looked up its docs to understand how it was leveraged. I first understood that we were linking the two fields using this lazy attribute, as if they were linked by foreign keys in the DB. 

But actually, I see that `primary_email` in the `FirefoxAccount` model is not linked to `email.primary_email` at all, they can be different AFAIU. So scratch that